images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -653,25 +653,27 @@ do
         # We added a timeout of 10 minutes (120 retries) before exit
         SERVICE_BROKER_COUNTER=1
         SERVICE_BROKER_TIMEOUT=180
-        until kubectl get --insecure-skip-tls-verify -n ${NAMESPACE} secret ${SERVICE_NAME}-operator-credentials
+        # use the secret name from the consumer to prevent credential clash
+        SECRET_NAME=$(kubectl --insecure-skip-tls-verify -n ${NAMESPACE} get mariadbconsumer/${SERVICE_NAME} -o yaml | shyaml get-value spec.secret)
+        until kubectl --insecure-skip-tls-verify -n ${NAMESPACE} get secret ${SECRET_NAME}
         do
         if [ $SERVICE_BROKER_COUNTER -lt $SERVICE_BROKER_TIMEOUT ]; then
           let SERVICE_BROKER_COUNTER=SERVICE_BROKER_COUNTER+1
-          echo "Secret ${SERVICE_NAME}-operator-credentials not available yet, waiting for 5 secs"
+          echo "Secret ${SECRET_NAME} not available yet, waiting for 5 secs"
           sleep 5
         else
-          echo "Timeout of $SERVICE_BROKER_TIMEOUT for ${SERVICE_NAME}-operator-credentials reached"
+          echo "Timeout of $SERVICE_BROKER_TIMEOUT for ${SECRET_NAME} reached"
           exit 1
         fi
         done
         # Load credentials out of secret
-        kubectl get --insecure-skip-tls-verify -n ${NAMESPACE} secret ${SERVICE_NAME}-operator-credentials -o yaml > /kubectl-build-deploy/lagoon/${SERVICE_NAME}-operator-credentials.yml
+        kubectl --insecure-skip-tls-verify -n ${NAMESPACE} get secret ${SECRET_NAME} -o yaml > /kubectl-build-deploy/lagoon/${SERVICE_NAME}-credentials.yml
```

Comment:
oooh using `${SERVICE_NAME}-credentials` instead of `${SERVICE_NAME}-operator-credentials.yml` is a great idea! <3